### ISSUE TITLE
[Bug]: PeoplePickerItemSuggestion is not accessible in Aquatic mode (contrast ratio)

### ISSUE THREAD

**Issue by tanneruzzell-msft** [microsoft/fluentui#34351]
### Package

common-styles

### Package version

8.118.2

### React version

17.0.2

### Environment

```shell
Having error running this command
```

### Current Behavior

![Image](https://github.com/user-attachments/assets/3f794a28-073d-434a-8d1f-19127c7fa49d)

### Expected Behavior

When user hovers over the people Picker, the low contrast makes it difficult to see.  There should be a more clear contrast ratio. Currently, the contrast ratio for hovering text is 1.169:1 which is less than 4.5:1 .

### Reproduction

https://developer.microsoft.com/en-us/fluentui#/controls/web/peoplepicker

### Steps to reproduce

1. Go to site above: https://developer.microsoft.com/en-us/fluentui#/controls/web/peoplepicker
2. Go to Normal People Picker
3. Change display to aquatic mode
4. Apply css: hover
5. see issue

![Image](https://github.com/user-attachments/assets/8bdafa74-bc8b-4fae-9ce0-1983be81b32c)

### Are you reporting an Accessibility issue?

yes

### Suggested severity

High - No workaround

### Are you willing to submit a PR to fix?

no

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.

**Comment by CampbellOwen** [comment]
When you apply real hover, the background is given `highlight` and the text content is `HighlightText` which are the proper high contrast colours to use here.

![Image](https://github.com/user-attachments/assets/53ef5e52-3928-49c2-875e-079c55a809ab)

![Image](https://github.com/user-attachments/assets/91d4aaef-1185-4b2c-ae80-23173b511020)

![Image](https://github.com/user-attachments/assets/230463c7-4fc4-4e34-ba23-39dadc25a110)

Are you seeing this when using actual hover or just by toggling `:hover` in the devtools?

**Comment by mskeyansong** [comment]
For the PeoplePicker issue, it only happens when we just use PeoplePickerItemSuggestion component. When we actually hover on it, we can see this issue. I attached more screenshots here. 
Our component when we apply real hover:

![Image](https://github.com/user-attachments/assets/0550e08b-e9b1-4cd4-be6f-065de20d1e5c)

![Image](https://github.com/user-attachments/assets/223bc907-e1f3-4f47-821b-30d1fa5a8713)

![Image](https://github.com/user-attachments/assets/4b75b7a8-6b9d-431b-b6cf-a2c7fae83e24)

![Image](https://github.com/user-attachments/assets/50c76749-5a13-41de-aad0-d90cea3a35bf)

![Image](https://github.com/user-attachments/assets/eee45169-6a9a-4b4d-89d7-0ad0bf2b5e19)

![Image](https://github.com/user-attachments/assets/8b25c920-4688-4ae7-96c0-e3f3ef27aa5a)
